docs/standard/modelling/identifiers.rst

```diff
@@ -1,142 +1,94 @@
+In BODS, real world identifiers can be published for:
 
-To create a link between statements, and the real-world organisations and people they relate to, statements may include a range of identifying information. We use a common :any:`Identifier object <schema-identifier>` with the following properties:
+- entities
+- people
+- stock markets and trading exchanges
+- tradable securities
 
-* ``scheme`` should be a value from a codelist of known identifier sources. Separate codelists exist for entities and persons. See below.
+Entity identifiers
+-------------------------------
 
-* ``id`` should be the value assigned to the relevant entity or person in that scheme.
+Use an :ref:`Identifier <schema-identifier>` in the ``identifiers`` array of an Entity Statement to supply a company registration number, Legal Entity Identifier (LEI), or other real world identifier.
 
-* ``uri`` may be used to provide a canonical URI for the entity or person within the scheme.
+The value for ``Identifier.scheme`` SHOULD come from `org-id.guide <http://org-id.guide>`_. This resource contains details of company registers and other identifier sources. If the identifier scheme you need to reference is not already listed on org-id.guide `propose a new entry <https://org-idguide-handbook.readthedocs.io/en/latest/contribute/#proposing-a-new-entry>`_.
 
-* ``schemeName`` should be the name of the list, registry or ID system.
+If it is useful to publish an internal identifier from your data management system, first consider whether it should be used as a :ref:`record identifier <record-identifiers>`. If not, either:
 
-A good-quality Identifier will contain ``scheme`` and ``id`` values which will uniquely identify an entity or person. Where these are not available, ``schemeName`` can be used to refer to the registration system in which the person or entity is known to be represented. When publishing an Identifier object, a value for either ``scheme`` or ``schemeName`` MUST be present.
+- publish your full list of internal identifiers, and propose it as an entry to `org-id.guide <http://org-id.guide>`_, or
+- use ``schemeName`` to identify your system and leave ``scheme`` blank
 
-Other identifiable objects
-++++++++++++++++++++++++++
+Person identifiers
+----------------------------------
 
-Links to identifiable objects, artefacts and institutions are also made elsewhere in the schema. For example, tradable company securities can be referenced with an identifier. More information on such elements of the schema is provided below.
+Global identifiers
++++++++++++++++++++
 
+If the system holds identification numbers for a person, and these can be published without privacy or security risks, then these SHOULD be included in the ``identifiers`` array of the Person Statement.
```

Comment:
```suggestion
If the system holds identification numbers for a person, and these can be published, then these SHOULD be included in the ``identifiers`` array of the Person Statement.
```

Reply:
the privacy risks are addressed further down so could shorten this a bit?

Reply:
I'm going to leave this bit of repetition in only because it sometimes isn't something that an implementer has thought much about. So we can't really say it enough!